app.rb

```diff
@@ -1,23 +1,18 @@
 require 'nypl_ruby_util'
 
-require_relative 'lib/s3_manager'
 require_relative 'lib/sqlite_manager'
```

Comment:
I'd rename this class/module. Since we're not using sqlite any more the name is misleading. I think it hypothetically could be eliminated as well, I'll leave some more comments below with my thoughts on this.